packages/manager/src/managers/project/ProjectManager.ts

```diff
@@ -50,9 +61,20 @@ export class ProjectManager extends BaseManager {
 				{ startDir: this.cwd },
 			);
 		} catch (error) {
-			throw new Error(
-				`Could not find a ${SLICE_MACHINE_CONFIG_FILENAME} file. Please create a config file at the root of your project.`,
-			);
+			const maybeDeprecatedConfigPath =
+				await this.getSliceMachineDeprecatedConfigPath();
+			if (maybeDeprecatedConfigPath) {
+				const newConfigPath = path.join(
+					this.cwd,
+					SLICE_MACHINE_CONFIG_FILENAME,
+				);
+				fs.rename(maybeDeprecatedConfigPath, newConfigPath);
+				this._cachedSliceMachineConfigPath = newConfigPath;
+			} else {
+				throw new Error(
+					`Could not find a ${SLICE_MACHINE_CONFIG_FILENAME} file. Please create a config file at the root of your project.`,
+				);
+			}
```

Comment:
As stated above, I'd avoid adding any kind of "inline" migration here, with time this would quickly get the codebase messy